live/live.default.xml

```diff
@@ -285,7 +285,22 @@
 
             </extractor>
 
-            <extractor name="org.dbpedia.extraction.mappings.AbstractExtractor" status="ACTIVE">
+            <extractor name="org.dbpedia.extraction.mappings.AbstractExtractor" status="PURGE">
```

Comment:
Thanks for that information. Do you have any suggestion how we should deal with that OAI-based AbstractExtractor? Our goal was to disable it in order to use the new AbstractExtractorWikipedia accessing the online Wikipedia directly. 

Should it stay enabled (similar to the OAI Feeders) or removed from the default config (maybe by adding a comment above the new AbstractExtractorWikipedia explaining how to switch back to the OAI-enabled AbstractExtractor)?